src/components/mixins/OrderBookMixin.ts

```diff
@@ -0,0 +1,23 @@
+import { FPNumber } from '@sora-substrate/util';
+import { Vue, Component } from 'vue-property-decorator';
+
+import { getter } from '@/store/decorators';
+
+import type { OrderBook } from '@sora-substrate/liquidity-proxy';
+
+@Component
+export default class OrderBookMixin extends Vue {
+  @getter.orderBook.currentOrderBook currentOrderBook!: Nullable<OrderBook>;
+
+  get pricePrecision(): number {
+    return (
+      this.currentOrderBook?.tickSize?.toLocaleString()?.split(FPNumber.DELIMITERS_CONFIG.decimal)?.[1]?.length ?? 0
```

Comment:
This will lead to a sort of unexpected behavior cuz `toLocaleString` depends on locale. This approach should definitely be changed

Reply:
It should be safe since decimals is either . or , and FPNumber.DELIMITERS_CONFIG.decimal is equal to . or ,    depends on locale

It also could be       
this.currentOrderBook?.tickSize?.toString()?.split('.')?.[1]?.length ?? 0

Reply:
10 -> 0
1
0.1 -> 1
0.001 -> 3